AnnoyancesFilter/Popups/sections/popups_allowlist.txt

```diff
@@ -44,6 +44,8 @@ g123.jp,accounts.binance.com#@##credential_picker_container
 !
 @@||my.leadpages.net/template/$script,domain=weddingjams.com|thesaturdaypaper.com.au|careercontessa.com
 @@||my.leadpages.net^*/public/
+! https://github.com/AdguardTeam/AdguardFilters/issues/177450
+@@||shop.blu-blumen.de^$script,xmlhttprequest
```

Comment:
Too wide. You need to unblock only the necessary url, and add a rule to a filter, which causes the problem.

Reply:
made changes

Reply:
@Alex-302 if it is good to go, the approve will be fine)